code/game/gamemodes/cult/items/tome.dm

```diff
@@ -90,9 +90,15 @@
 		if(use_check_and_message(scribe))
 			return
 
-		scribe.visible_message(SPAN_CULT("[scribe] slices open their palm with a ceremonial knife, drawing arcane symbols with their blood..."))
-		playsound(scribe, 'sound/weapons/bladeslice.ogg', 50, FALSE)
-		scribe.drip(4)
+		scribe.visible_message(SPAN_WARNING("[scribe] closes their eyes and presses their hand against a page in their book, then takes out a piece of chalk from the book and draws strange symbols on the ground..."), SPAN_CULT("You close your eyes and worshipfully pierce your hand against the spikes on the rune's page, imbuing the tome's chalk with its magic and your blood.  You feel His presence wash over you, sealing the wounds as you begin drawing with the chalk."))
```

Comment:
I'm not sure "worshipfully" is a word
There's also an extra space after "blood."

I think this entire visible message is WAY too verbose anyway. It needs to be more compact and to the point. The chalk is also a bit of a magic item that you can never see, which I personally dislike.

Reply:
Fixed!